examples/utils/example-utils/src/modelLoader/modelLoader.ts

```diff
@@ -56,14 +56,18 @@ export class ModelLoader<ModelType> implements IModelLoader<ModelType> {
 	// Here we specifically pick just the loader props we know we need to keep API exposure low.  Fine to add more
 	// here if we determine they're needed, but they should be picked explicitly (e.g. avoid "scope").
 	public constructor(
-		props: Pick<ILoaderProps, "urlResolver" | "documentServiceFactory" | "codeLoader"> & {
+		props: Pick<
+			ILoaderProps,
+			"urlResolver" | "documentServiceFactory" | "codeLoader" | "logger"
```

Comment:
We should be sure to note in the PR description that we have added the ability to provide a logger to `ModelLoader` in `example-utils`.